x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/add_source/add_source_list.tsx

```diff
@@ -97,12 +98,22 @@ export const AddSourceList: React.FC = () => {
     filterConfiguredSources
   ) as SourceDataItem[];
 
+  const Layout = isOrganization ? WorkplaceSearchPageTemplate : PersonalDashboardLayout;
```

Comment:
This same logic gets repeated in 3 places - SourceLayout, AddSource, and SourceList - might be worth DRYing out if y'all plan on adding more source pages at some point 🤷

Reply:
Agree, added it to the polish list.